packages/gasket-utils/README.md

```diff
@@ -8,9 +8,9 @@ Reusable utilities for Gasket internals
 npm i @gasket/utils
 ```
 
-## Usage
+<!-- ## Usage
```

Comment:
Is this supposed to be commented?

Reply:
@agerard-godaddy What do you think is required here, in terms of documentation?

Reply:
I'd be fine with removing the usage section for this package for now.